Create.c

```diff
@@ -296,8 +296,7 @@ static int add_disk_to_super(int mdfd, struct shape *s, struct context *c,
 		remove_partitions(fd);
 	if (st->ss->add_to_super(st, &info->disk, fd, dv->devname,
 				 dv->data_offset)) {
-		ioctl(mdfd, STOP_ARRAY, NULL);
```

Comment:
ioctl call must stay unless you can prove that it is necessary.

Reply:
I put it back... I haven't proven that it is unnecessary.